packages/Modal/stories/Modal.stories.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, boolean, select } from "@storybook/addon-knobs";
+import styled from "styled-components";
+import Button from "@paprika/button";
+import SidePanel from "@paprika/sidepanel";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Modal from "../src";
+
+/* Long block to test body scroll locking */
+const LongBlock = styled.div`
+  padding-bottom: 200vh;
+`;
+
+const DemoFullWidthContent = styled.div`
+  background-image: linear-gradient(to bottom, #79ff85, #70b3ff);
+  height: 1000px;
+`;
+
+const ModalStory = ({ children }) => {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const toggle = () => {
+    setIsOpen(state => !state);
+  };
+
+  return (
+    <LongBlock>
+      <Button onClick={toggle}>Open</Button>
+      <Modal isOpen={isOpen} onClose={toggle} size={select("Size", ShirtSizes.DEFAULT, ShirtSizes.MEDIUM, "Modal")}>
+        <Modal.Header hasCloseButton={boolean("Has close button", true, "Modal.Header")}>Header</Modal.Header>
+        {children}
+        <Modal.Footer>
+          <Button kind="primary">Primary</Button>
+          <Button>Secondary</Button>
+          <Button kind="minor" onClick={toggle}>
+            Cancel
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    </LongBlock>
+  );
+};
+
+storiesOf("Modal", module)
+  .addDecorator(withKnobs)
+  .add("Basic", () => (
+    <ModalStory>
+      <Modal.Content>
+        {Array(25)
```

Comment:
Maybe this helper could be extended to include some custom content?

https://github.com/acl-services/paprika/blob/master/.storybook/assets/styles/common.styles.js#L57